src/Diagnostics.ModelsAndUtils/Attributes/ResourceFilter/ArmResourceFilter.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Diagnostics.ModelsAndUtils.Attributes
```

Comment:
nitpick : For new files, lets do the documentation as mandated. :)

Reply:
Updated documentation for the files that I am adding. :-)